network/p2p/acp118/handler_test.go

```diff
@@ -143,18 +142,31 @@ func TestHandler(t *testing.T) {
 
 // The zero value of testVerifier allows signing
 type testVerifier struct {
-	Errs []*common.AppError
+	t            *testing.T
+	errs         []*common.AppError
+	clientNodeID ids.NodeID
+}
+
+func newTestVerifier(t *testing.T, clientNodeID ids.NodeID, errs []*common.AppError) *testVerifier {
+	return &testVerifier{
+		t:            t,
+		errs:         errs,
+		clientNodeID: clientNodeID,
+	}
 }
```

Comment:
nit: This is a matter of preference but I don't like adding these constructor methods unless give us something (e.g add initialization logic or something), but since these are just wiring parameters I'd prefer to not have this